docs/team/liwen.adoc

```diff
@@ -0,0 +1,61 @@
+= Lai Liwen - Project Portfolio
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+== PROJECT: ProgressChecker
+
+---
+
+== Overview
+
+ProgressChecker is *a desktop application for CS2103/T students to have a centralized hub that tracks their learning progress and project development* throughout the module. The current release is meant to be used alongside with CS2103/T module website. ProgressChecker has a JavaFX GUI, but users will interact with it via CLI for efficiency.
+
+== Summary of contributions
+
+* *Major enhancement*: added *the ability to upload profile photo*
+** *What it does:* allows students to upload any photos as their own or teammates's profile photo.
+** *Justification:* this feature solves the issue that students may have teammates with the same name.Profile photo can help students to recognize each other better.
+** *Highlights:* This enhancement touches on all 4 components in the code base. It required an in-depth analysis of the existing architecture on how each components work with one another in order to implement it in an optimal way.
```

Comment:
In justification section, there's a full stop error :( 
Just add a space after name. :)